packages/core/src/components/icon/icon.story.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+import { c, classy } from '../../utils';
+
+/**
+ * `.ods-icon` requires SVG sprite to be included in your app. If you wish to
+ * use individual SVGs, import them directly from `@onfido/castor-icons/svg`
+ * instead.
+ *
+ * https://github.com/onfido/castor-icons
+ */
+export const Icon = ({ name }: IconProps) =>
+  html('svg', {
+    class: classy(c('icon')),
+    fill: 'currentColor',
```

Comment:
I believe you're missing "color" change here:

```suggestion
export const Icon = ({ name, color: token }: IconProps) =>
  html('svg', {
    class: classy(c('icon')),
    fill: token ? color(token) : 'currentColor',
```

Reply:
Nice catch!